stories/FAutocompleteInput.stories.ts

```diff
@@ -0,0 +1,161 @@
+import { Story } from '@storybook/vue3';
+import FAutocompleteInput, {
+  FAutocompleteInputProps,
+} from '@/components/form/FAutocompleteInput.vue';
+import FButton from '@/components/FButton.vue';
+import { ref } from 'vue';
+
+export default {
+  title: 'Components/Form/FAutocompleteInput',
+  component: FAutocompleteInput,
+};
+
+const Template = (args: FAutocompleteInputProps) => ({
+  components: { FAutocompleteInput },
+  setup: () => ({ args }),
+  template: '<FAutocompleteInput v-bind="args" />',
+});
+
+const capitals = [
+  { label: 'Amsterdam', description: 'Capital of Netherlands', value: 'amsterdam' },
+  { label: 'Berlin', description: 'Capital of Germany', value: 'berlin' },
+  { label: 'Madrid', description: 'Capital of Spain', value: 'madrid' },
+  { label: 'Rome', value: 'rome' },
+  { label: 'London', value: 'london' },
+  { label: 'Paris', description: 'Capital of France', value: 'paris' },
+  { label: 'Prague', description: 'Capital of Czech Republic', value: 'prague' },
+  { label: 'Stockholm', description: 'Capital of Sweden', value: 'stockholm' },
+  { label: 'Vienna', value: 'vienna' },
+];
+
+export const Default: Story<FAutocompleteInputProps> = Template.bind({});
+Default.args = {
+  placeholder: 'Search capitals...',
+  options: capitals,
+  errorMessage: 'Select a value in the list to validate',
+};
+
+export const WithLabel: Story<FAutocompleteInputProps> = Template.bind({});
+WithLabel.args = {
+  placeholder: 'Search capitals...',
+  options: capitals,
+  label: 'Choose a capital',
+  errorMessage: 'Select a value in the list to validate',
+};
+
+export const WithHint: Story<FAutocompleteInputProps> = Template.bind({});
+WithHint.args = {
+  placeholder: 'Search capitals...',
+  options: capitals,
+  hint: `You can navigate in the menu through keyboard`,
+  hintIcon: 'infoCircle',
+  errorMessage: 'Select a value in the list to validate',
+};
+
+export const Full: Story<FAutocompleteInputProps> = Template.bind({});
+Full.args = {
+  placeholder: 'Search capitals...',
+  options: capitals,
+  label: 'Choose a capital',
+  hint: `You can navigate in the menu through keyboard`,
+  hintIcon: 'infoCircle',
+  errorMessage: 'Select a value in the list to validate',
+};
+
+export const Error: Story<FAutocompleteInputProps> = Template.bind({});
+Error.args = {
+  placeholder: 'Search capitals...',
+  options: capitals,
+  label: 'Choose a capital',
+  validateOnMount: true,
+  errorMessage: 'Choose an option to dismiss the error',
+  rules: value => value !== '',
+};
```

Comment:
Does this story works ?

Reply:
It does !
and actually the added rule is not needed here to make it work, I'll remove it.